src/Runner.php

```diff
@@ -178,17 +183,61 @@ public function run($input = null, $output = null, $app = null, $commandFiles =
         return $statusCode;
     }
 
+    public function setLoader(ClassLoader $loader)
+    {
+        return $this->setCommandFilePluginPrefixes($loader->getPrefixesPsr4());
+    }
+
+    public function setCommandFilePluginPrefixes($prefixes)
+    {
+        $this->commandFilePluginPrefixes = $prefixes;
+        return $this;
+    }
+
+    public function setCommandFilePluginPattern($namespacePattern)
+    {
+        $this->namespacePattern = $namespacePattern;
+        return $this;
+    }
+
+    protected function getPluginCommandClasses()
+    {
+        $commandClasses = [];
```

Comment:
Would you mind making this method public? That way, an application that leverages robo can call it and build its own command classes. You may also want to consider making getRoboFileCommands()  public as well.